SPECS/xcp-ng-linstor.spec

```diff
@@ -1,11 +1,12 @@
 Summary: Install all LINSTOR dependencies
 Name: xcp-ng-linstor
-Version: 0.9.0
```

Comment:
Why going from a 3 number version to a 2?

Reply:
Basically only the minor version interested me and I probably wanted to use the patch to add few fixes later during the beta but in fact not. Major/minor is enough.